### DISCUSSION TITLE
Minus sign is flipped in unit 1.4 video 3

### DISCUSSION THREAD

**Discussion by muhaksim** [Lightning-AI/dl-fundamentals#10] [Lecture videos and quizzes] [answered]
While I was watching video 3 from unit _1.4 The First Machine Learning Classifier_, I couldn't understand how x2 is 1.14. I think it should be a negative value and not a positive one since it is lower than zero in the diagram. You can see the photo at 01:12 of the video.

What do you think? Correct me if I am wrong about this.

![Screenshot (185)](https://user-images.githubusercontent.com/91332164/208222206-45dbb9f0-5cc6-4a89-9cad-d20bc4ef0926.png)

**Comment by rasbt** [comment, marked as answer]
Good catch, thanks! This should have been -1.14. I will try to fix that in the video!

**Reply by muhaksim** [reply]
Sebastian, thanks for the simple way you explain concepts in the course. It is opening my appetite for deep learning. I am a recent CS grad from Somalia.

**Reply by rasbt** [reply]
I am really glad to hear that you are finding it useful! Thanks for sharing, that's very motivating to hear! Happy learning!!

PS: New units will be coming out soon (there will be 10 units in total). Unit 3 got a bit hung up in the editing process but is hopefully out either next week or after the holidays.
